play-zipkin-tracing/core/src/main/scala/brave/play/ZipkinTraceServiceLike.scala

```diff
@@ -170,7 +170,8 @@ trait ZipkinTraceServiceLike {
       (carrier: A, key: String) => getHeader(carrier, key).orNull
     ).extract(headers)
 
-    tracer.joinSpan(contextOrFlags.context())
+    Option(contextOrFlags.context())
```

Comment:
this is good, just need test case. you can use something like only sampled header to test.

ex. test that `X-B3-Sampled: 0` results in a new unsampled trace

Reply:
It seems like I would need https://github.com/openzipkin-contrib/play-zipkin-tracing/pull/32 to write a test for this?

Reply:
unit test could be done now I suspect.